pages.it/common/node.md

```diff
@@ -0,0 +1,24 @@
+# node
+
+> Piattaforma JavaScript Server-side (Node.js).
+> Maggiori informazioni: <https://nodejs.org>.
+
+- Esegue un file JavaScript:
+
+`node {{percorso/del/file}}`
```

Comment:
In Italian is the preposition `to` could be rendered with `al` (preposition `a` + article `il`) in the case of the movement of something toward something else.
In the colloquial way, eventually, it is more widespread the usage of `del` (preposition `di` + article `il`) for this purpose (in English would be _path **of** file_).

The last option is also semantically more correct and accurate in Italian, in fact other files also make use of it (https://github.com/tldr-pages/tldr/blob/main/pages.it/common/more.md, https://github.com/tldr-pages/tldr/blob/main/pages.it/common/age.md, ..)

source: I am Italian

Reply:
@ffranchina we use `allo`/`al`/`alla` for paths in all Italian pages. It's better to follow the existing convention rather than mix and change it across pages. See for example: [cp](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/cp.md), [7z](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/7z.md), [chsh](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/chsh.md).

Reply:
It will give me shivers to do so but, if this is the policy we want to follow, so it be :)

Reply:
@ffranchina I see your point my friend :') I think for now it is better to follow the convention. A PR can always be open to discuss and change all occurrences from `al` to `del` in one go. Should be doable with some scripting and manual checks. If you decide to do that mention me in the PR.

Reply:
@mebeim Great to hear that, I'll propose such change then! :D